app/views/i_belong_mailer/auto_enrolled.html.erb

```diff
@@ -15,37 +16,38 @@
+    Did you know that completing the Encouraging girls into GCSE computer science course means you're already on the way to achieving the I Belong certificate for your school?
   </p>
   <p class='govuk-body'>
-    Now you're enrolled, you will be able to access your personal dashboard, where you can find out more about the certificate and the next steps required to achieve it.  
+    Now you're enrolled, you will be able to access your personal dashboard, where you can find out more about the certificate and the next steps required to achieve it.
   </p>
 
   <p class='govuk-body'>
     <%= link_to 'Explore activities in your dashboard', i_belong_url, class: 'govuk-button button' %>
   </p>
 
   <p class='govuk-body'>
-    Has your school already completed some of the activities required to achieve the certificate? Tick them off by submitting evidence in your dashboard.  
+    Has your school already completed some of the activities required to achieve the certificate? Tick them off by submitting
+    <%= link_to 'evidence', static_asset_url('Structuring+your+evidence+for+I+Belong.docx') %> in your dashboard.
   </p>
- 
+
   <p class='govuk-body'>
-    Ideas to get you started 
+    Ideas to get you started
   </p>
   <ul class="govuk-list govuk-list--bullet">
-  	<li>Maximise our recommended key stage 3 curriculum resources, aligned to evidence-based approaches supporting girls' engagement in computer science. 
-    Examples include 
-    <%= link_to 'Clear messaging in digital media, Developing for the Web', '/curriculum/key-stage-3/clear-messaging-in-digital-media' %>, 
-    and 
-    <%= link_to 'Media - Animations', 'https://teachcomputing.org/curriculum/key-stage-3/media-animations' %>.</li>
     <li>Provide access for students to a computing-related STEM lunchtime or after-school club and show how your approach is gender responsive. You and your colleagues may be doing this already and be ready to submit your evidence.</li>
+  	<li>Maximise our recommended key stage 3 curriculum resources, aligned to evidence-based approaches supporting girls' engagement in computer science.
+    Examples include
+    <%= link_to 'Clear messaging in digital media, Developing for the Web', 'https://teachcomputing.org/curriculum/key-stage-3/clear-messaging-in-digital-media' %>,
+    and
+    <%= link_to 'Media - Animations', 'https://teachcomputing.org/curriculum/key-stage-3/media-animations' %>.</li>
   </ul>
- 
+
   <p class='govuk-body'>
     <strong>Need more support?</strong>
   </p>
- 
+
   <p class='govuk-body'>
-    Guidance is available in the <%= link_to 'programme handbook', static_asset_url('I+Belong+Handbook.pdf') %> to help you and your colleagues choose meaningful activities for your school. 
+    Guidance is available in the <%= link_to 'programme handbook', static_asset_url('I+Belong+Handbook.pdf') %> to help you and your colleagues choose meaningful activities for your school.
```

Comment:
Swap to helper method for handbook (might be worth flagging we are linking to one handbook, but we are going to have two?)